src/Transport/Connectivity/NamespaceManagerAdapter.cs

```diff
@@ -113,9 +114,15 @@ public Task<TopicDescription> GetTopic(string path)
             return manager.GetTopicAsync(path);
         }
 
+        // TODO: delete this once NamespaceManagerAdapter is internalized
         public Task DeleteSubscriptionAsync(string topicPath, string subscriptionName)
         {
-            return manager.DeleteSubscriptionAsync(topicPath, subscriptionName);
+            return DeleteSubscription(new SubscriptionDescription(topicPath, subscriptionName));
+        }
+
+        public Task DeleteSubscription(SubscriptionDescription subscriptionDescription)
```

Comment:
Why is the delete subscription contained in this PR? I suppose those changes are unrelated to this PR?